test/qtismtest/data/content/enums/RoleTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
```

Comment:
can we add return typehint on these methods?

Reply:
done